ykman/cli/otp.py

```diff
@@ -131,6 +131,24 @@ def info(ctx):
     controller = ctx.obj['controller']
     click.echo(dev.device_name)
     slot1, slot2 = controller.slot_status
+
+    if dev.is_fips:
+        fips_mode = controller.is_in_fips_mode
+        summary = fips_mode and 'Yes' or 'No'
```

Comment:
Is `Yes/No` a summary? Maybe this will be easier to follow if click.echo is called directly if fips mode, and the different explanations in the else branch?

Reply:
I think my main reason for using only one echo call was to keep it all on one line of output. But I agree the clarity here can be improved, I have some ideas for that.

Reply:
Potential solution proposed in https://github.com/Yubico/yubikey-manager/pull/125

Reply:
Fixed in #125